### ISSUE TITLE
FR: Supercharge WYSIWYG mode

### ISSUE THREAD

**Issue by HEmile** [mdelobelle/obsidian_supercharged_links#54]

**Comment by sehlhorst** [comment]
I ran into a cpu-spike when typing if the plugin (v 0.3.5) is running in the new build 0.13 version (even with wysiwyg mode _disabled_).  by disabling obsidian_supercharged_links plugin, the problem goes away.

**Comment by mdelobelle** [comment]
I'm not experiencing this issue with my vault, but maybe it's not the same size or number of files as yours.
could you give us some information about number of files? 
does the spike occur when you are editing a large note as well as a small one?
could you make a screenshot of your settings for this plugin?
could you try to disable some settings one by one (backlinks, file explorer, inline fields....)?

**Comment by sehlhorst** [comment]
here's a short video of the cpu spike https://www.youtube.com/watch?v=Udp-9OEOC60 (pls let me know after you've seen it so I can disable it - it is an unlisted link.

**Comment by sehlhorst** [comment]
That particular file was ~1300 words.  
In a file with 50 words, cpu spikes to "moderate" but not to "very high" power usage
I have a total of 2500 files
enable/disable of "show in editor" is what causes / ends the spike

here's the settings - 
![image](https://user-images.githubusercontent.com/44043184/141379190-f39bb3fc-e1b7-46b7-b074-161b25d45687.png)

**Comment by mdelobelle** [comment]
I've seen the video thank you. I'll try to reproduce the issue on my vault

**Comment by brimwats** [comment]
also a +1 to supercharge Live Preview mode

**Comment by HEmile** [comment]
I will likely pick this up when Live Preview is stable enough for me to use it above CodeMirror options :p

**Comment by HEmile** [comment]
@mdelobelle I made it work for live preview, but it doesn't work for aliased links... Not sure if there's an easy workaround for that since there's no attr on the link to get that info from.
If you want you can make a release for this.

**Comment by HEmile** [comment]
Should also work for aliased links now.

**Comment by brimwats** [comment]
Unsure if this is related to an update: 

1) links are not clickable in live preview.
2) the stylizations (tag pills) on live preview are constantly turning off and on

**Comment by HEmile** [comment]
1. Hmm, I'm getting this too in the latest insider but it's not a supercharged links thing (I tried without). 
2. Yes, this is a bit of a drawback that I couldn't find a workaround for yet. I'll try to improve this when I think the editor is more stable.

**Comment by brimwats** [comment]
1. good to know!
2. yes absolutely wait! no impatience in my part, just want to file them so y'all can track them! 😁

**Comment by HEmile** [comment]
Fair enough! In general, I'm afraid supercharging in live preview will be pretty buggy when doing the emoji thing unfortunately :( Editing when using that seems to be a mess from what I tried. But colours work fine.

**Comment by HEmile** [comment]
This works almost perfectly in 0.4.0. Current limitations:
- Doesn't work with DataView Inline links (See #29 )
- Incompatibilities with Mobile. We'll probably wait for a new Obsidian version

**Comment by brimwats** [comment]
I propose that this could be closed and a new issue for mobile compatibility could be opened!
